src/test/resources/data/StudentProfilePageUiTest.json

```diff
@@ -77,24 +107,60 @@
         "sectionLevel": {},
         "sessionLevel": {}
       }
+    },
+    "instructor1OfTestingSanitizationCourse": {
+      "googleId": "SHomeUiT.sanitizationInstruct",
+      "courseId": "SHomeUiT.sanitizationCourse",
+      "name": "Instructor<script> alert('hi!'); </script>",
+      "email": "SHomeUiT.instruct@sanitization.tmt",
+      "isArchived": false,
+      "role": "Co-owner",
+      "isDisplayedToStudents": true,
+      "displayedName": "inst<script>alert('hi!');</script>",
+      "privileges": {
+        "courseLevel": {
+          "canviewstudentinsection": true,
+          "cangivecommentinsection": true,
+          "cansubmitsessioninsection": true,
+          "canmodifysessioncommentinsection": true,
+          "canmodifycommentinsection": true,
+          "canmodifycourse": true,
+          "canviewsessioninsection": true,
+          "canmodifysession": true,
+          "canviewcommentinsection": true,
+          "canmodifystudent": true,
+          "canmodifyinstructor": true
+        },
+        "sectionLevel": {},
+        "sessionLevel": {}
+      }
     }
   },
   "students": {
     "studentWithExistingProfile": {
       "googleId": "${value.injected.in.testfile}",
       "email": "${value.injected.in.testfile}",
       "course": "SHomeUiT.CS2104",
-      "name": "Benny C</option></td></div>'\"",
-      "comments": "This student's name is BC</option></td></div>'\"",
-      "team": "Team 1</td></div>'\""
+      "name": "Benny C",
+      "comments": "This student's name is BC",
+      "team": "Team 1"
     },
     "SHomeUiT.benny.c@SHomeUiT.CS2104": {
       "googleId": "SHomeUiT.benny.c",
       "email": "SHomeUiT.benny.c@gmail.tmt",
       "course": "SHomeUiT.CS2104",
       "name": "Benny Charles",
       "comments": "This student's name is Benny Charles",
-      "team": "Team 1</td></div>'\""
+      "team": "Team 1"
+    },
+    "student1InTestingSanitizationCourse": {
+      "googleId": "student1InTestingSanitizationCourse",
+      "email": "SHomeUiT.stud@sanitization.tmt",
+      "course": "SHomeUiT.sanitizationCourse",
+      "name": "Stud1<script> alert('hi!'); </script>",
+      "comments": "<script> alert('hi!'); </script>",
+      "team": "Team tags&\"</td>",
```

Comment:
Any reason why it must be like that? I believe this test case does not test whether the tags are properly closed, only whether the value is properly sanitized before being displayed.